src/components/ReportActionItemIOUPreview.js

```diff
@@ -47,20 +49,31 @@ const propTypes = {
         // Currently logged in user email
         email: PropTypes.string,
     }).isRequired,
+
+    ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     iou: {},
+    iouReportID: undefined,
+    shouldHidePayButton: false,
+    onPayButtonPressed: null,
 };
 
 const ReportActionItemIOUPreview = ({
-    action,
-    isMostRecentIOUReportAction,
-    hasOutstandingIOU,
     iou,
     personalDetails,
     session,
+    shouldHidePayButton,
+    onPayButtonPressed,
+    translate,
 }) => {
+    const sessionEmail = lodashGet(session, 'email', null);
+
+    // Pay button should be visible to manager person in the report
```

Comment:
"to manager person" this reads a bit funny and feels like it's missing a word or something. Is this saying that the button should only be visible to the report manager?

Reply:
Yep, will rewrite this.